src/pdm/userservicedesk/HRService.py

```
@@ -21,6 +23,15 @@ class HRService(object):
     """
     _logger = logging.getLogger(__name__)
 
+    @staticmethod
+    @startup
+    def load_userconfig(config):
+        """ Configure the HRService application.
+            Gets the key needed to contact the Credential Service
+        """
+        current_app.cs_key = config.pop("CS_secret", "XkjgscdJ")
```

This should fail if the secret key is not set rather than use a pre-set default.